app/vmagent/main.go

```diff
@@ -322,11 +325,16 @@ func requestHandler(w http.ResponseWriter, r *http.Request) bool {
 			w.Write([]byte("OK"))
 		}
 		return true
+	default:
+		if strings.HasPrefix(r.URL.Path, "/static") {
+			http.FileServer(http.FS(staticFiles)).ServeHTTP(w, r)
```

Comment:
Probably, it's better to move into variable:

var staticServer = http.FileServer(http.FS(staticFiles))

 and use it later as:

staticServer.ServerHTTP(w,r)  

@hagen1778  WDYT?

Reply:
Yes, that looks reasonable.